test/language/statements/for-in/identifier-let-allowed-as-lefthandside-expression-not-strict.js

```diff
@@ -1,11 +1,12 @@
 // Copyright (C) 2011 the V8 project authors. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 13.6.0.1
 description: >
-    for declaration:
     identifier "let" allowed as lefthandside expression
+esid: sec-iteration-statements
+info: |
+  for ( [ lookahead ∉ { let [ } ] LeftHandSideExpression [?Yield, ?Await] in 
+    Expression[+In, ?Yield, ? Await]) Statement[?Yield, ?Await, ?Return]
 flags: [noStrict]
 ---*/
-for (let in {}) { }
-
+for (let in {}) { }
```

Comment:
Missing a new line character here, but I'm not going to block on that.

Reply:
my editor extensions are not up to date.